pub-critic/src/app/components/navigation/Navigation.module.scss

```diff
@@ -0,0 +1,118 @@
+.Navigation {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    width: 100%;
+    position: fixed;
+    top: 0;
+    padding: 20px 5vw;
+    background-color: black;
+    z-index: 100;
+    height: 70px;
+    @media (max-width: 768px) {
+        padding: 20px 15px;
+    }
+    .Name {
```

Comment:
`camelCase` so start with small letter

Reply:
Honestly I thought that PascalCase was fine for module classes, is it acceptable or should I change it?